core/src/main/resources/config.yml

```
@@ -63,7 +63,8 @@ join:
 
 # Changes various gameplay mechanics.
 gameplay:
-  refill-wool: true # Should wool in wool rooms be automatically refilled?
+  refill-wool: true              # Should wool in wool rooms be automatically refilled?
+  global-legacy-flag-beams: true # Should everyone see legacy flag beams?
```

Looks a bit on the long side, maybe just `flag-beams`?
Should probably move to the `ui` section instead of `gameplay` ?
I think false as a default makes more sense, as this is supposed to be for legacy (1.7) clients

```suggestion
  flag-beams: false # Should everyone see floating wool flag beams?
```

---

I think flag-beams gives off more of a should I have any flag beams at all, thought I agree it is kind of long, but just `flag-beams` is not descriptive enough in my opinion.

The `ui` section seems reasonable.

I'm open to changes if anyone has a shorter key.

---

`ui.flag-beams` is reasonable

---

Decided to just go to `ui.flag-beams`